src/transformation/builtins/console.ts

```diff
@@ -16,6 +16,7 @@ export function transformConsoleCall(
     const parameters = transformArguments(context, expression.arguments, signature);
 
     switch (methodName) {
+        case "error":
```

Comment:
Maybe also add the remaining console API functions like log, warn ...

https://console.spec.whatwg.org/#console-namespace

Reply:
Good call, added .info and .warn too